lib/Test/Nginx/Socket.pm

```diff
@@ -3920,6 +3921,18 @@ only see the logs for the last test run (which you usually do not control
 except if you set C<TEST_NGINX_NO_SHUFFLE=1>). With this, you accumulate
 all logs into a single file that is never cleaned up by Test::Nginx.
 
+=head2 TEST_NGINX_ARCHIVE_PATH
+
+Archive files and output of test blocks, which start up nginx server and
```

Comment:
The word "output" here is quite ambiguous here since there are so many different kinds of "output", like test comparison output, nginx error log output, http responses, and special outputs from special testing mode like the valgrind test mode. Better be more explicit here. I think you just mean the http response data from nginx.